.classpath

```diff
@@ -42,8 +42,8 @@
 	<classpathentry kind="var" path="SAFS_BUILD/libs/autoit/AutoItX4Java.jar"/>
 	<classpathentry kind="var" path="SAFS_BUILD/libs/autoit/jacob.jar"/>
 	<classpathentry kind="var" path="SAFS_BUILD/libs/slf4j-api-1.7.21.jar"/>
-	<classpathentry kind="var" path="SAFS_BUILD/libs/httpclient5/httpclient5-5.0-alpha1.jar" sourcepath="/HTTPCLIENT5SOURCE"/>
-	<classpathentry kind="var" path="SAFS_BUILD/libs/httpclient5/httpcore5-5.0-alpha1.jar" sourcepath="/HTTPCORE_SOURCE"/>
+	<classpathentry exported="true" kind="var" path="SAFS_BUILD/libs/httpclient5/httpclient5-5.0-alpha2-SNAPSHOT.jar" sourcepath="/HTTPCLIENT5SOURCE"/>
+	<classpathentry exported="true" kind="var" path="SAFS_BUILD/libs/httpclient5/httpcore5-5.0-alpha2.jar" sourcepath="/HTTPCORE_SOURCE"/>
```

Comment:
The httpclient and core jars are exported so they are on the classpath for test projects that depend on the project in Eclipse.